worlds/shivers/docs/en_Shivers.md

```diff
@@ -23,9 +23,9 @@ If the player receives a key then the corresponding door will be unlocked. If th
 
 ## What is the victory condition?
 
-Victory is achieved when the player captures Lightning in the generator room.
+Victory is achieved when the player has captured the required number Ixupi set in options.
```

Comment:
"set in options" is a bit odd, you probably want "their options"
```suggestion
Victory is achieved when the player has captured the required number Ixupi set in their options.
```

Reply:
Changed.